_includes/figure.liquid

```diff
@@ -14,13 +16,17 @@
     {% if site.imagemagick.enabled %}
       <source
         class="responsive-img-srcset"
-        srcset="{% for i in site.imagemagick.widths %}{{ img_path | relative_url }}-{{ i }}.webp {{i}}w,{% endfor %}"
+        {% if ext == '.jpg' or ext == '.jpeg' or ext == '.png' or ext == '.tiff' or ext == '.gif' %}
```

Comment:
Since the path was splitted with `.`, I believe that the `.` will not be part of the `ext` anymore. Have you tested this?

Reply:
I have tested it, the code works as expected